Makefile

```diff
@@ -136,7 +136,9 @@ source:
 pkg-clean:
 	$(RM) -r $(DESTDIR)
 
-rpm dpkg: pkg-clean source
+rpm dpkg: pkg-clean
+	mkdir -p $(DESTDIR)/pmdk
+	find . -maxdepth 1 -not -name $$(basename $(DESTDIR)) -not -name .git -not -name . -exec cp -r "{}" $(DESTDIR)/pmdk \;
```

Comment:
You don't check whether DESTDIR is empty.

Please extract the code that lives under "make source" to a script and merge it with what you did here when current directory is not a git repository.